### PR TITLE
Add support for guzzle 6

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -7,11 +7,13 @@
     "license": "Apache-2.0",
     "require": {
         "php": ">=5.4",
-        "google/auth": "~0.4",
+        "google/auth": "dev-add-support-for-guzzle-6",
```
Comment (bshaffer) [inline composer.json]: this will be updated to a tagged version once we have one

Comment (jkenn99) [inline composer.json]: 0.5 is tagged and supports guzzle 6

Comment (bshaffer) [inline composer.json]: done

**Comment by austinpray** [comment]
I was getting 

```
 google/apiclient dev-master requires guzzlehttp/guzzle 5.2.* -> no matching package found.
```

requiring this branch via

```
"google/apiclient": "dev-add-support-for-guzzle-6#305fd9c43ccc7921ab71cd848b0420c3e8f620fd"
```

Fixes the issue. Thanks for this!

**Comment by bshaffer** [comment]
anybody who was using this composer branch can now checkout "google/apiclient": "~2.0@RC"

**Comment by austinpray** [comment]
Thanks guys!

**Pull request merged**